project.mml

```diff
@@ -1525,6 +1528,7 @@ Layer:
             OR "natural" IN ('peak', 'volcano', 'saddle', 'spring', 'cave_entrance')
             OR historic IN ('memorial', 'monument', 'archaeological_site', 'wayside_cross')
             OR tags->'memorial' IN ('plaque')
+            OR tags->'emergency' IN ('phone')
```

Comment:
This [should be `tags @> 'emergency=>phone'`](https://github.com/gravitystorm/openstreetmap-carto/blob/master/CONTRIBUTING.md#sql-style-guidelines), as well as the repetition in the WHERE clause for the other layer.

Reply:
Hm, but why should we check if a tag is in the hstore? Why do we need this special construction at all?

Reply:
If we don't include an item in the OR part of the WHERE clause it won't be selected in the layer.

Reply:
The icon on a node was present in my test rendering, so it looks for me that was selected.

Reply:
The problem is `OR tags->'emergency' IN ('phone')` does not follow our style standards.

Reply:
I see. Does `OR tags->'memorial' IN ('plaque')` have this problem too?

Reply:
Yes, but you don't need to change it in this PR, it's an existing error.